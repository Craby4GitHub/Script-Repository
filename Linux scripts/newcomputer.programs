audacity
bat
build-essential
cmatrix
curl
dash
dcfldd
deja-dup
detox
dosfstools
etherwake
exa
fail2ban
fatresize
figlet
fish
flameshot
fzf
gdebi
git
gnome-boxes
gnome-tweaks
gparted
htop
imagemagick
ksh
libreoffice
lolcat
neofetch
nmap
pandoc
parallel
peazip
pigz
pv
prettyping
python3-pip
qdirstat
remmina
remmina-plugin-rdp
remmina-plugin-secret
sed
smartmontools
tcsh
terminology
toilet
traceroute
unrar
virt-manager
vlc
whois
xclip
zsh
