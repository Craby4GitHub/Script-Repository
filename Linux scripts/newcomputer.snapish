code
cool-retro-term
crystal
docfetcher
gitkraken
krita
libreoffice
nvim
pinta
powershell
uno-calculator
