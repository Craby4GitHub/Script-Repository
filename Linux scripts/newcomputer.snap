code --classic
cool-retro-term --classic
crystak --classic
docfetcher
gitkraken --classic
krita
libreoffice
nvim --classic
pinta
powershell --classic
uno-calculator --beta
